browser/ui/webui/brave_webui_source.cc

```diff
@@ -11,6 +11,7 @@
 #include "brave/components/ipfs/browser/buildflags/buildflags.h"
 #include "base/strings/utf_string_conversions.h"
 #include "brave/common/url_constants.h"
+#include "brave/components/moonpay/browser/buildflags/buildflags.h"
```

Comment:
make sure to add a dep for the buildflag in brave/uil/BUILD.gn

Reply:
fixed